### PR TITLE
MIRA 4.0.2 is GPLv2+ and also available on Mac OS X

### PR DESCRIPTION
**Pull request by peterjc**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Following existing recipe approach and using the pre-compiled binaries from MIRA author Bastien Chevreux for either 64 bit Linux or Mac OS X (Darwin).

### CODE REVIEW

File: recipes/mira/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-  license: ''
+  license: 'GNU General Public License v2 or later (GPLv2+)'
+  license_file: LICENCE
   summary: 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)'
 
 build:
-  number: 1
-  skip: True # [osx]
+  number: 2
+  skip: True # [not linux64 or osx]
```
Comment (bgruening) [inline recipes/mira/meta.yaml]: You are still skipping here OSX, this is not what you want isn't it?
Just remove this line.

Comment (peterjc) [inline recipes/mira/meta.yaml]: Well spotted - I was making assumptions about the operator precedence, and should probably have written ``# [not linux64 and not osx]`` here?

If currently BioConda only supports Mac OS X and 64 bit Linux removing the conditional would be fine for now.

Presumably we don't do 32 bit Linux, nor anything interesting like ARM or ...?

Comment (bgruening) [inline recipes/mira/meta.yaml]: Yes, we only do linux64 and osx at the moment.

**Comment by jerowe** [comment]
sorry, this has been sitting for awhile.

Do we have to use static binaries? Those seem to be troublesome more often than not.

As for the libgcc*dylib, I have no idea. I have no experience with OSX. Maybe @bioconda/osx will have some ideas?

**Comment by peterjc** [comment]
The Mac recipe fails with:

```
Fixing linking of /usr/lib/libSystem.B.dylib in /anaconda/conda-bld/mira_1497001237721/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin/mira
Fixing linking of @executable_path/../lib/libgcc_s.1.dylib in /anaconda/conda-bld/mira_1497001237721/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/bin/mira
Error: Could not find libgcc_s.1.dylib
```

I though including ``libgcc`` as a run time dependency would have solved this, but it did not.

We could go down the route of compiling MIRA, but from knowledge of multiple queries on the MIRA mailing list of the potential pitfalls I was not inclined to try this myself when the author's binaries have previously just worked.

I could refocus this pull request on just Linux if that is expedient (fixing license, checksums, etc)?

**Comment by jerowe** [comment]
@peterjc, there are a lot of assemblers with complex steps I have never been able to get working on osx. With the biocontainers project it is not that big of a deal anymore. If someone really wants to run their software from a mac they can use the biocontainer.

**Comment by jerowe** [comment]
I contributed the original recipe, and I would have tried to compile it from source. You may of course have better luck. ;-)

**Comment by cbrueffer** [comment]
This needs some updating due to 5f052908d6f9d4412836cf10e652f8ec06788ea0 ; sorry, was unaware of this PR.

**Comment by peterjc** [comment]
@cbrueffer do you agree the license is GPL v2 or later, not just GPL v2?

**Comment by cbrueffer** [comment]
You're right, I stopped reading the docs after "MIRA has been put under the GPL version 2.".

**Comment by peterjc** [comment]
Rebased, if the Mac changes still fail I will remove those, and just focus on the license clarification.

**Pull request merged**